script/c16480084.lua

```diff
@@ -0,0 +1,51 @@
+--エヴォルダー・ペルタ
+local s,id=GetID()
+function s.initial_effect(c)
+    local e1=Effect.CreateEffect(c)
+    e1:SetDescription(aux.Stringid(69633792,0))
+    e1:SetCategory(CATEGORY_ATKCHANGE)
+    e1:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_F)
+    e1:SetCode(EVENT_SPSUMMON_SUCCESS)
+    e1:SetCondition(aux.evospcon)
+    e1:SetOperation(s.evolreg)
+    c:RegisterEffect(e1)
+end
+function s.evolreg(e)
+    local c=e:GetHandler()
+    --defup
+    local e1=Effect.CreateEffect(c)
+    e1:SetType(EFFECT_TYPE_SINGLE)
+    e1:SetCode(EFFECT_UPDATE_DEFENSE)
+    e1:SetValue(500)
+    e1:SetReset(RESET_EVENT+RESETS_STANDARD+RESET_DISABLE)
+    c:RegisterEffect(e1)
+    --search
+    local e2=Effect.CreateEffect(c)
+    e2:SetDescription(aux.Stringid(id,0))
+    e2:SetCategory(CATEGORY_TOHAND+CATEGORY_SEARCH)
+    e2:SetType(EFFECT_TYPE_SINGLE+EFFECT_TYPE_TRIGGER_O)
+    e2:SetCode(EVENT_BATTLE_DESTROYED)
+    e2:SetCondition(s.schcon)
+    e2:SetTarget(s.schtg)
+    e2:SetOperation(s.schop)
+    e2:SetReset(RESET_EVENT+0x7b0000)
```

Comment:
in case you want to remove magic number
0x7b0000 = RESETS_STANDARD - (RESET_DISABLE + RESET_TURN_SET + RESET_LEAVE + RESET_TOFIELD)

Reply:
Updated
thank you